src/components/RedirectToNextGenDialog.svelte

```diff
@@ -0,0 +1,52 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+ 
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import { t } from '../i18n';
+  import {
+    hasSeenRedirectDialog,
+    setAsSeenRedirectDialog,
+  } from '../script/stores/complianceStore';
+  import StandardDialog from './dialogs/StandardDialog.svelte';
+  import StandardButton from './StandardButton.svelte';
+
+  let isOpen = !hasSeenRedirectDialog();
+  const redirectToNextGen = () => {
+    setAsSeenRedirectDialog();
+    window.location.href = 'https://ml.microbit.org';
+  };
+  const onClose = () => {
+    setAsSeenRedirectDialog();
+    isOpen = false;
+  };
+</script>
+
+<StandardDialog
+  {isOpen}
+  hasCloseButton={false}
+  closeOnOutsideClick={false}
+  closeOnEscape={false}
+  class="w-100 space-y-5"
+  {onClose}>
+  <svelte:fragment slot="heading">
+    {$t('popup.redirectToNextGen.header')}
```

Comment:
I'd be tempted to name these more generically for easy porting to the other branch which just has different text/URLs.

No strong view, it's all temporary.

Reply:
E.g. 
AppVersionRedirectDialog
popup.appVersionRedirect.explain etc.
might make it easy to pull across and flip.

Reply:
That makes sense, I have renamed them to follow your suggestions